File: C++/Problem282A_Bit++.cpp
#include <iostream>
#include <cstring>
using namespace std;

int main(){
    int loop_count;
    cin >> loop_count;

    string input;
    int result=0;
    while (loop_count--){
        cin >> input;
        if(input == "++X" || input == "X++")
            result++;
        else
            result--;
    }
    cout << result;
    return 0;
}

File: Python/Problem282A_Bit++.py
def main():
    loop_count = int(input())
    output = 0
    while loop_count:
        loop_count-=1
        line = input()
        if line == "++X" or line == "X++":
            output += 1
        else:
            output -= 1
    print(output)

if __name__ == "__main__":
    main()